website/i18n/en.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
Did you create it? If yes, mind adding a space after colons, to make the JSON better? :)